app/src/main/res/values/colors.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <color name="purple_200">#FFBB86FC</color>
-    <color name="purple_500">#FF6200EE</color>
-    <color name="purple_700">#FF3700B3</color>
+    <color name="purple_500">#33691E</color>
```

Comment:
You changed the color code but not the color name. Not good, not good! 😄 

Now you have a color purple which is actually not purple but green 😄

Reply:
DONE 😄